nativescript-angular/bin/update-app-ng-deps

```diff
@@ -13,6 +13,13 @@ const isNgDependency = name => name.startsWith("@angular") ||
     name === "zone.js" ||
     name === "typescript";
 
+const update = (deps, dep) => {
+    if (deps[dep.dependencyName]) {
```

Comment:
This will prevent the script from adding `@angular/platform-browser-dynamic` to the dependencies object.